docs-content/general/6_product-features/6_metrics/dashboard-management.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Dashboard Management
+slug: dashboard-management
+---
+
+## Dashboard overview
+
+At [app.highlight.io/metrics](https://app.highlight.io/metrics), there is an overview of all dashboards in your project. Here, you can create or delete dashboards, or click into the dashboard you want to view / edit. If your project has many dashboards, you can filter on their name using the search.
+
+By default, all projects have an "Insights" dashboard automatically created to show useful graphs and showcase many of the highlight.io metrics features.
+
+## Viewing / editing a dashboard
+
+Within a dashboard, there are a few controls available at the top of the page:
+- **Date picker** adjusts the queried time range for all graphs on the page, even those that don't show any timestamps. When you copy a dashboard's URL, the time range is persisted in that URL.
+- **Edit dashboard** toggles edit mode to allow you to update the dashboard's name or drag and drop graphs to reorganize the dashboard. Using the "Cancel" button to exit edit mode will revert any changes made.
+- **Add graph** opens the graph editor to create a new graph. You can read more [here](./graphing.md).
+
+There are also controls available per graph, shown when hovering over each graph in a dashboard:
+- **Expand graph** opens a larger version of the graph for more detail.
+- **Edit graph** opens the graph editor to edit an existing graph. You can read more [here](./graphing.md).
```

Comment:
Can we put an image of this at the top of this section? (just below the title)

Reply:
https://www.dropbox.com/scl/fi/pn9lra931nzmljpme6bb1/dashboard_view.png?rlkey=ru5b2sae167nbb0uuq02onmvh&dl=0